integration/test/performancetest/performance_query_utils.go

```diff
@@ -198,6 +198,12 @@ func GetPerformanceMetrics(instanceId string, agentRuntime, logNum, tps int, age
 
 	//add actual test data with statistics
 	for _, result := range metrics.MetricDataResults {
+		//convert memory bytes to MB
+		if (*result.Label == "procstat_memory_rss") {
+			for i, val := range(result.Values) {
+				result.Values[i] = val / (1000000)
```

Comment:
Should this be 1024 * 1024 = 1048576?

Reply:
I believe they are both valid depending on the context, but for most things `1000^2` is used. https://en.wikipedia.org/wiki/Megabyte
And then for some things like operating systems, 1024^2 is used. 🤷 
But we decided to use MB because it is a more meaningful measure of memory used and easier to know at a glance roughly how much memory the agent is using rather than trying to count how many zeros there are to see if it is 1 MB, 10MB or 100MB, etc.